docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,123 @@
+
+The issue is demonstrated in the following animation:
+
+![Head of line blocking][animation] 
+
+This animation shows that if a certain packet drops in transmission, all packets have to wait at the transport layer until it is resent by the other end.
+If the dropped packet is resent and arrived then all packets are freed from the transport layer. 
+
+Let's look at two areas where this head-of-line blocking issue is a huge deal.
+
+**Multiplayer Game Networking**
+
+Multiplayer action games are based on a constant stream of packets sent at a speed ranging from 10 to 30 packets per second.
+For the most part, the data in these packages are so time-sensitive that only the most recent data is useful.
+You can think of the input and position of the player, the orientation and speed, and the state of the physical objects in the world.
+If a single packet drops out we can not afford to queue up 10-30 packets a second until the lost packet is retransmitted. 
+This could cause annoying lag behavior and a bad user experience. 
+
+**Web Networking**
+
+Game networking is not the only area where this head-of-line blocking is a big issue.
+The World Wide Web is a place where quick web-page load speeds are important (who wants to wait 200ms to the long right?).
+As websites get bigger and attention decreases, we need faster loading times for websites.
+
+To tackle this issue, HTTP-2 introduced a technique called multiplexing. 
+In short, this means that multiple TCP streams are initialized to communicate with the server. 
+Then If one of them blocks the whole website can continue to load seemingly while that single stream is retransmitting.
+
+We will take a deeper dive into this subject when looking at QUIC multiplexing.
+    
+### Connection Setup Duration
+
+In the standard HTTP+TLS+TCP stack, TCP needs a handshake to establish a session between server and client, 
+and TLS needs its handshake to ensure that the session is secured.
+
+![TCP-handshake](./images/tcp-handshake.svg.png)
+
+First, the source sends an 'SYN initial request' packet to the target server to start the dialogue. 
+Then the target server sends an 'SYN-ACK packet' to agree to the process.
+Lastly, the source sends an 'ACK packet' to the target to confirm the process, after which the message exchange can start. 
+ 
+Now if we want to secure the TCP connection, we have to use a protocol like TLS on top of it. 
+In the case of TLS versions older than 1.3, an additional three more handshake messages are required.
+
+You can see how expensive it is to create a secure TCP connection. 
+In a scenario of TCP and TLS 1.2 with a 100ms latency, we need to wait for 6 x 100ms = 600ms to set up a connection. 
+If the website is big, an additional load time can make the website load over a second. 
+This, of course, is disturbing for our short attention spans. 
```

Comment:
I'm skeptical this is a big win compared to TCP + TLS 1.3 and all the same features.

Reply:
I did state older than 1.3, but still with TCP + TLS 1.3 we need 4 round trips in total if I'm correct? The point I want to get at with this section is to be able to refer it later if we compare the QUIC connection setup with the TCP connection setup.